docs/en/guide/deploy.md

```diff
@@ -166,13 +186,19 @@ Don't enable options like _Auto Minify_ for HTML code. It will remove comments f
          - name: Setup Pages
            uses: actions/configure-pages@v4
          - name: Install dependencies
-           run: npm ci # or pnpm install / yarn install / bun install
+           run: npm install # or pnpm install / yarn install / bun install
```

Comment:
Is there a reason to use npm install instead of npm ci?

If npm ci gives you troubles although locally your package lock is in sync then you likely need to disable legacy-peer-deps in ~/.npmrc on your local machine, then rm -rf node_modules and rm package-lock.json then npm install again.

see: https://stackoverflow.com/a/76400582/3098783

Might be a good addition as comment in case people struggle with this errir